src/constants/projects.ts

```diff
@@ -0,0 +1,30 @@
+import { INotifyProjectWithComingSoon } from '@/utils/types'
+
+export const COMING_SOON_PROJECTS: Array<INotifyProjectWithComingSoon> = [
```

Comment:
I don't think this info should be part of the codebase of this repo. Feels wrong to me to hardcode data in source code like this. Also we can just update that JSON file in the other repo, without needing to touch Web3Inbox?

Can we make a new GitHub repo and put a static JSON file there and `fetch()` it?

Reply:
This is a temporary solution (at least I hope), why maintain this basic logic in another repo? Also, it wouldn't be a big deal but there will be load time from the GH or somewhere else while fetching, which is slower than this approach.

The reason I'd chose fetching from another source would be that hardcoded solution doesn't seem quite professional.